packages/blockchain-extension/test/commands/packageSmartContractCommand.test.ts

```diff
@@ -940,6 +973,61 @@ describe('packageSmartContract', () => {
             logSpy.should.have.been.calledTwice;
         });
 
+        it('should package if GOPATH contains multiple paths and one of them is right for the project', async () => {
+            await createTestFiles('goProject', '0.0.1', 'golang', true, false);
+
+            const testIndex: number = 2;
+            workspaceFoldersStub.returns(folders);
+            showWorkspaceQuickPickStub.resolves({
+                label: folders[testIndex].name,
+                data: folders[testIndex]
+            });
+
+            findFilesStub.withArgs(new vscode.RelativePattern(folders[testIndex], '**/*.go'), null, 1).resolves([vscode.Uri.file('chaincode.go')]);
+
+            showInputStub.onFirstCall().resolves('myProject');
+            showInputStub.onSecondCall().resolves('0.0.3');
+            showInputStub.onThirdCall().resolves('myProject');
+            showInputStub.onCall(3).resolves('0.0.3');
+
+            process.env.GOPATH = '/' + path.delimiter + testWorkspace;
```

Comment:
Similar to the previous suggestion: change '/' to a path, instead of just one character, say /some/path/'. It's not wrong, but forces the reader to look again and pause, so it takes longer to understand the test.